app/gui2/src/components/visualizations/TableVisualization.vue

```diff
@@ -161,7 +161,7 @@ const agGridOptions: Ref<GridOptions & Required<Pick<GridOptions, 'defaultColDef
   onFirstDataRendered: updateColumnWidths,
   onRowDataUpdated: updateColumnWidths,
   onColumnResized: lockColumnSize,
-  copyHeadersToClipboard: true,
+  copyHeadersToClipboard: false,
```

Comment:
`false` is a default, so it could be just removed (unless we keep default values here as well, for some reason).